mne/evoked.py

```diff
@@ -501,6 +500,8 @@ def animate_topomap(self, ch_type=None, times=None, frame_rate=None,
 
             .. versionadded:: 0.22
         %(verbose)s
+            .. versionadded:: 1.1.0
+        %(vmin_vmax_topomap)s
```

Comment:
in fact, my earlier comment was not quite correct (sorry!).  I forgot to mention that if there is a `verbose` param it should always come last.  So please move `vmin`/`vmax` to just before `verbose`.  Also the `.. versionadded::` bit should come *after* the parameter it's referring to, so switch the order of these two lines please (and also move them to before `verbose`).